### DISCUSSION TITLE
Help with GitHub webhook to reviewboard

### DISCUSSION THREAD

**Discussion by rongitone** [community/community#23006] [New to GitHub]
<p>Hi, I have repository setup at github and it’s got a webhook to reviewboard 3.0 and I receive a 200 OK response from GitHub when testing.<br>
The reviewboard server is on premise.</p>
<p>I have followed the directions here <a href="https://www.reviewboard.org/docs/manual/3.0/admin/configuration/repositories/github/" rel="nofollow noopener">https://www.reviewboard.org/docs/manual/3.0/admin/configuration/repositories/github/</a><br>
which seem to indicate once the webhook is inserted in the github repository everything will work.</p>
<p>I would like to have reviewboard send out a review request with a link to a diff when a person has made a commit/push to the github repository.<br>
I would also like to have reviewboard send out a notice when someone has reviewed a commit on reviewboard.</p>
<p>With a Git repository there is a .git/hooks directory where the hooks are located and called. With GitHub<br>
Is there a directory where github hooks are located on the reviewboard server so when a commit/push is made they are called?</p>
<p>Does the <a href="http://mydomain.com" rel="nofollow noopener">mydomain.com</a> url below need to reference the webhook by name?</p>
<p>The payload url from reviewboard is this according to the instructions.<br>
<a href="https://mydomain.com/reviewboard/repos/3/github/hooks/close-submitted/" rel="nofollow noopener">https://mydomain.com/reviewboard/repos/3/github/hooks/close-submitted/</a></p>
<p>Instructions say to include the information below when closing a commit.<br>
Review Request <span class="hashtag">#123</span></p>
<p>I make a commit and push it to github. The commit succeeds but that’s all. It’s not clear how to get a review from it.<br>
Thanks, Ron</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>Thanks for reaching out.</p>
<p>If you need help configuring reviewboard, you may want to contact <a href="https://www.reviewboard.org/support/" rel="nofollow noopener">their support resources</a> instead. We don’t have any experience configuring reviewboard and won’t be able to help with that.</p>
<p>From your description, it sounds like the webhook is being sent properly, so I’m not sure what else we can do there.</p>
<p>The <a href="https://git-scm.com/book/en/v2/Customizing-Git-Git-Hooks" rel="nofollow noopener"><code>.git/hooks</code> directory</a> is not the same thing as <a href="https://developer.github.com/webhooks/" rel="nofollow noopener">GitHub webhooks</a>. The git hooks feature allows you to add code to customize the behavior of a git repository in specific ways. GitHub webhooks notify external services of specific repository events so those services can do something based on that input. The two features can be used in similar ways. On the other hand, they can be used in very dissimilar ways.</p>
<p>I hope that helps. Let us know if you have more questions.</p>
